content/influxdb/v2.4/admin/internals/tsi/rebuild-index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Rebuild the TSI index
+description: >
+  Flush and rebuild the TSI index to purge corrupt index files or remove indexed
+  data that is out of date. 
+menu:
+  influxdb_2_4:
+    parent: Manage TSI indexes
+weight: 201
+related:
+  - /influxdb/v2.4/reference/internals/storage-engine/
+  - /influxdb/v2.4/reference/internals/file-system-layout/
+  - /influxdb/v2.4/reference/cli/influxd/inspect/build-tsi/
+---
+
+In some cases, it may be necessary to flush and rebuild the TSI index.
+For example, purging corrupt index files or removing indexed data that may be
+out of date.
```

Comment:
```suggestion
In some cases, it may be necessary to flush and rebuild the TSI index.
For example, purging corrupt index files or removing outdated indexed data.
```